mozilla-release/mobile/android/base/java/org/mozilla/gecko/controlcenter/SiteTrackersListAdapter.java

```diff
@@ -480,11 +480,12 @@ private int calculateTotalTrackerCount(GeckoBundle categoryBundle) {
 
     private int calculateBlockedTrackerCount(GeckoBundle categoryBundle) {
         int blockedTrackerCount = 0;
-        final String pagehost = data.getBundle("data").getBundle("summary").getString("pageHost");
+        final String pagehost = GeckoBundleUtils.safeGetString(data, "data/summary/pageHost");
         final List<String> blacklist = Arrays.asList(GeckoBundleUtils.safeGetStringArray(data, "data/summary/site_blacklist"));
         final List<String> whitelist = Arrays.asList(GeckoBundleUtils.safeGetStringArray(data, "data/summary/site_whitelist"));
+        final boolean isGhosteryPaused = GeckoBundleUtils.safeGetBoolean(data, "data/summary/paused_blocking");
         final GeckoBundle[] trackers = categoryBundle.getBundleArray("trackers");
-        if (whitelist.contains(pagehost)) {
+        if (whitelist.contains(pagehost) || isGhosteryPaused) {
```

Comment:
Could you reorder the lines on the function to avoid doing somethings that we ignore.
I recommend
```
 final boolean isGhosteryPaused = GeckoBundleUtils.safeGetBoolean(data, "data/summary/paused_blocking");
if (isGhosteryPaused) {
            return 0;
}
final String pagehost = GeckoBundleUtils.safeGetString(data, "data/summary/pageHost");
final List<String> whitelist = Arrays.asList(GeckoBundleUtils.safeGetStringArray(data, "data/summary/site_whitelist"));
if (whitelist.contains(pagehost)){
           return 0;
}
int blockedTrackerCount = 0;
final List<String> blacklist = Arrays.asList(GeckoBundleUtils.safeGetStringArray(data, "data/summary/site_blacklist"));
final GeckoBundle[] trackers = categoryBundle.getBundleArray("trackers");
....
....
.....
```